components/calendar/src/indian.rs

```diff
@@ -198,7 +198,7 @@ impl Calendar for Indian {
     fn year(&self, date: &Self::DateInner) -> types::YearInfo {
         types::YearInfo::new(
             date.0.year,
-            types::EraYear::new(tinystr!(16, "saka"), date.0.year),
+            types::EraYear::new_single_era(tinystr!(16, "saka"), date.0.year),
```

Comment:
I don't understand what `single` refers to in this context.

Reply:
Renamed to new_with_sole_era

Reply:
Again, what? Are there `EraYear`s with more than one era?

Reply:
It's to say this is a _calendar_ with a single era.

Reply:
I think I'll just remove this, we added this to smooth the transition to formatting vs standard era, now it serves a less strong purpose.